pom.xml

```diff
@@ -163,6 +163,27 @@
           <version>3.2.19.0</version>
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
+          <version>2.15.2</version>
+        </dependency>
+        <dependency>
+          <groupId>com.fasterxml.jackson.dataformat</groupId>
+          <artifactId>jackson-dataformat-yaml</artifactId>
+          <version>2.15.2</version>
+        </dependency>
+        <dependency>
+          <groupId>com.fasterxml.jackson.module</groupId>
+          <artifactId>jackson-module-scala_${scala.binary.version}</artifactId>
+          <version>2.15.2</version>
+        </dependency>
+        <!-- Paranamer dependency required by jackson-module-scala -->
+        <dependency>
+          <groupId>com.thoughtworks.paranamer</groupId>
+          <artifactId>paranamer</artifactId>
+          <version>2.8</version>
+        </dependency>
```

Comment:
This transient dependency can be removed unless your code calls into this library directly.